tests/src/test/spark330/scala/org/apache/spark/sql/rapids/suites/RapidsJsonFunctionsSuite.scala

```diff
@@ -20,6 +20,23 @@ spark-rapids-shim-json-lines ***/
 package org.apache.spark.sql.rapids.suites
 
 import org.apache.spark.sql.JsonFunctionsSuite
+import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.rapids.utils.RapidsSQLTestsTrait
 
-class RapidsJsonFunctionsSuite extends JsonFunctionsSuite with RapidsSQLTestsTrait {}
+class RapidsJsonFunctionsSuite extends JsonFunctionsSuite with RapidsSQLTestsTrait {
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    SQLConf.get.setConfString("spark.rapids.sql.expression.JsonTuple", "true")
+    SQLConf.get.setConfString("spark.rapids.sql.expression.GetJsonObject", "true")
+    SQLConf.get.setConfString("spark.rapids.sql.expression.JsonToStructs", "true")
+    SQLConf.get.setConfString("spark.rapids.sql.expression.StructsToJson", "true")
+  }
+
+  override def afterAll(): Unit = {
+    super.afterAll()
+    SQLConf.get.unsetConf("spark.rapids.sql.expression.JsonTuple")
+    SQLConf.get.unsetConf("spark.rapids.sql.expression.GetJsonObject")
+    SQLConf.get.unsetConf("spark.rapids.sql.expression.JsonToStructs")
+    SQLConf.get.unsetConf("spark.rapids.sql.expression.StructsToJson")
+  }
```

Comment:
Consider extracting into a base trait and reuse conf handling across unit tests
https://github.com/NVIDIA/spark-rapids/blob/56de325aadaa097354fcd1b5ffef30921e5154ea/tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala#L158-L161

Reply:
will do